src/apps/tco2_dashboard/retirement_trends/impl/retirement_trends_chain.py

```diff
@@ -251,25 +263,31 @@ def modify_klima_token_retirements_df(self, df):
             df['Beneficiary'] + ')'
         )
         df['Date'] = df[
-            'Date'].astype(str).str.split(n=1)
+            'Date'].astype(str).str.split(n=1).str[0]
+        df['Project ID'] = df['Project ID'].str.split("-", expand=True)[1]
```

Comment:
This should include the full `VCS-123` format and it should link to the Verra project page like the other Retirement Trends pages do: https://registry.verra.org/app/projectDetail/VCS/985